options/processing_options.go

```diff
@@ -159,7 +162,7 @@ func NewProcessingOptions() *ProcessingOptions {
 		Blur:              0,
 		Sharpen:           0,
 		Dpr:               1,
-		Dither:            DitherOptions{Type: DitherNone, Contrast: false, Native: false, Desaturate: false},
+		Dither:            DitherOptions{Type: DitherNone, Contrast: false, Native: false, Desaturate: false, Meter13: false, Clamp: false},
```

Comment:
Do we want a default for `SoftProof` here?

Reply:
ya!